docs/man_pages/project/testing/livesync-ios.md

```diff
@@ -3,13 +3,14 @@ livesync ios
 
 Usage | Synopsis
 ------|-------
-General | `$ tns livesync ios [--device <Device ID>] [--watch]`
+General | `$ tns livesync ios [--device <Device ID>] [--watch] [--emulator]`
 
-Synchronizes the latest changes in your project to iOS devices. 
+Synchronizes the latest changes in your project to iOS devices or iOS simulator. 
```

Comment:
Run a check for this description (it should be appearing in the index.md and the Related Commands across the testing section). Update it in all instances to match the description here.